app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml

```diff
@@ -4,8 +4,8 @@
       .browse-ontology-title-bar
         %a.browse-ontology-title{:href => "/ontologies/#{ontology[:acronym]}", data: {'turbo-frame': '_top'}}
           = ontology[:name]+" ("+ontology[:acronym]+")"
-          - if ontology[:private]
-            %i.fas.fa-key{title:"Private Ontology"}
+          - ontology_status = status_string(ontology)
+          = render Display::InfoTooltipComponent.new(text: ontology_status, icon: submission_status_icons(ontology_status)) if session[:user]&.admin?
```

Comment:
you removed the private ontology icon here, put it again.

Reply:
Done, thank you for the comment ^^